controllers/tab/issueEntry/form/IssueEntryPublicationMetadataForm.inc.php

```diff
@@ -234,10 +234,17 @@ function execute($request) {
 
 			$user = $request->getUser();
 
+			// Get a list of author user IDs
+			$authorUserIds = array();
+			$stageAssignmentDao = DAORegistry::getDAO('StageAssignmentDAO');
+			$submitterAssignments = $stageAssignmentDao->getBySubmissionAndRoleId($submission->getId(), ROLE_ID_AUTHOR);
+			$submitterAssignment = $submitterAssignments->next();
+			assert($submitterAssignment); // At least one author should be assigned
```

Comment:
The payment stuff is in a terrible state right now -- mostly hidden from the interface and not well maintained. I've got that scheduled for attention for OJS 3.1 along with subscriptions.